src/core/common/settings.cpp

```diff
@@ -150,12 +150,11 @@ const char *SettingsBase::KeyToString(Key aKey)
         "",                  // (6)  kKeyReserved
         "SlaacIidSecretKey", // (7)  kKeySlaacIidSecretKey
         "DadInfo",           // (8)  kKeyDadInfo
-        "OmrPrefix",         // (9)  kKeyOmrPrefix
+        "BrUlaPrefix",       // (9)  kKeyBrUlaPrefix
         "OnLinkPrefix",      // (10) kKeyOnLinkPrefix
         "SrpEcdsaKey",       // (11) kKeySrpEcdsaKey
         "SrpClientInfo",     // (12) kKeySrpClientInfo
         "SrpServerInfo",     // (13) kKeySrpServerInfo
-        "Nat64Prefix",       // (14) kKeyNat64Prefix
```

Comment:
Should we try to erase the `Nat64Prefix` entry in the settings to free the space?

Reply:
NAT64 prefix is not enabled yet. So there should be no device using it.